casacore/tables/table.py

```diff
@@ -1796,19 +1795,18 @@ def browse(self, wait=True, tempname="/tmp/seltable"):
                 os.system('casabrowser ' + self.name() + waitstr1)
             elif len(tempname) > 0:
                 six.print_("  making a persistent copy in table " + tempname)
-                tx = self.copy(tempname)
-                tx = 0
```

Comment:
`tx` is a local variable and I don't see anywhere else in the method, it is being used.  Moreover, `tx` is assigned 0 immediately after it has been assigned `self.copy(tempname)` which makes the assignment of `self.copy(tempname)` irrelevant .Its value won't change dynamically, so,I removed it.